source/man/doveadm-compress-connect.1.rst.in

```diff
@@ -0,0 +1,37 @@
+.. _man-1_doveadm-compress-connect:
+
+========================
+doveadm-compress-connect
+========================
+
+NAME
+====
+
+doveadm-compress-connect - Establish a compress-aware imap connection
+
+SYNOPSIS
+========
+
+**doveadm compress-connect** *host* [*port*]
+
+DESCRIPTION
+===========
+
+**doveadm-compress-connect** connects to the given *host:port* service,
+and waits for the IMAP command **COMPRESS DEFLATE** to be casted.
+When that happens, it turns on client side compression/decompression
+of the socket streams, in order to follow server side switch and allow
+the protocol conversation to continue.
```

Comment:
this whole paragraph describes what happens in the code, which is probably not relevant for the user of this command, it should instead try to explain what it does from the user's perspective.